src/reducers/selectors.js

```diff
@@ -0,0 +1,114 @@
+// @flow
+
+import { createSelector } from 'reselect';
+
+export const EMPTY_OBJ = {};
+export const EMPTY_AR = [];
+
+// Checks a response object for errors and returns them
+export const processErrors = (responseValue: ?QueryResponse): Object => {
+	if (!responseValue || !responseValue.error) {
+		return {};
+	}
+	return responseValue;
+};
+
+export const processErrorsMaybe = (responseValue: ?QueryResponse) => {
```

Comment:
in `communicationSelectors`, there is a function that needs to return a falsey value if there are no errors, whereas `processErrors` always returns something truthy (`{}`). I'd like to see this go away just like `processErrors` to avoid the confusion.